src/commands/build/wranglerjs/mod.rs

```diff
@@ -164,3 +231,11 @@ pub fn install(cache: &Cache) -> Result<PathBuf, failure::Error> {
     run_npm_install(wranglerjs_path.clone()).expect("could not install wranglerjs dependencies");
     Ok(wranglerjs_path)
 }
+
+fn random_chars(n: usize) -> String {
```

Comment:
We could split this one out of Wranglerjs build command

Reply:
right now it's only used to generate the wranglerjs temp file. If anything, I would just put this next to where the temp file is created in wranglerjs

Reply:
Could we inline it in the temp_file generation?